Doc/library/stdtypes.rst

```diff
@@ -4370,8 +4370,8 @@ type, the :dfn:`dictionary`.  (For other containers see the built-in
 A dictionary's keys are *almost* arbitrary values.  Values that are not
 :term:`hashable`, that is, values containing lists, dictionaries or other
 mutable types (that are compared by value rather than by object identity) may
-not be used as keys.  Numeric types used for keys obey the normal rules for
-numeric comparison: if two numbers compare equal (such as ``1`` and ``1.0``)
+not be used as keys.
+If there are some values that compare equal (such as ``1``, ``1.0``, and ``True``)
 then they can be used interchangeably to index the same dictionary entry.  (Note
 however, that since computers store floating-point numbers as approximations it
 is usually unwise to use them as dictionary keys.)
```

Comment:
If wanted

```suggestion
then they can be used interchangeably to index the same dictionary entry.
```

Reply:
I agree with Mark that we can remove this note.